plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/KubernetesClusterManagerImpl.java

```diff
@@ -1147,6 +1147,14 @@ public boolean startKubernetesCluster(long kubernetesClusterId, boolean onCreate
         }
     }
 
+    private Account getOwnerOrCaller(KubernetesClusterVO kubernetesCluster) {
+        Account owner = accountService.getActiveAccountById(kubernetesCluster.getAccountId());
+        if (owner.getType() == Account.Type.PROJECT) {
```

Comment:
```suggestion
        if (owner == null || owner.getType() == Account.Type.PROJECT) {
```
@davidjumani maybe this and refactor `getServiceUserKeys`